docs/usage/llms/local-llms.mdx

```diff
@@ -6,57 +6,69 @@ description: When using a Local LLM, OpenHands may have limited functionality. I
-- 2025/03/31: We released an open model OpenHands LM v0.1 32B that achieves 37.1% on SWE-Bench Verified
+- 2025/03/31: We released an open model OpenHands LM 32B v0.1 that achieves 37.1% on SWE-Bench Verified
 ([blog](https://www.all-hands.dev/blog/introducing-openhands-lm-32b----a-strong-open-coding-agent-model), [model](https://huggingface.co/all-hands/openhands-lm-32b-v0.1)).
 
+## Quickstart: Running OpenHands with a Local LLM using LM Studio
 
-## Quickstart: Running OpenHands on Your Macbook
+This guide explains how to serve a local Devstral LLM using [LM Studio](https://lmstudio.ai/) and have OpenHands connect to it.
 
-### Serve the model on your Macbook
+We recommend:
+- **LM Studio** as the local model server, which handles metadata downloads automatically and offers a simple, user-friendly interface for configuration.
+- **Devstral Small 2505** as the LLM for software development, trained on real GitHub issues and optimized for agent-style workflows like OpenHands.
 
-We recommend using [LMStudio](https://lmstudio.ai/) for serving these models locally.
+### Hardware Requirements
 
-1. Download [LM Studio](https://lmstudio.ai/) and install it
+Running Devstral requires a recent GPU with at least 16GB of VRAM, or a Mac with Apple Silicon (M1, M2, etc.) with at least 32GB of RAM.
 
-2. Download the model:
-   - Option 1: Directly download the LLM from [this link](https://lmstudio.ai/model/devstral-small-2505-mlx) or by searching for the name `Devstral-Small-2505` in LM Studio
-   - Option 2: Download a LLM in GGUF format. For example, to download [Devstral Small 2505 GGUF](https://huggingface.co/mistralai/Devstral-Small-2505_gguf), using `huggingface-cli download mistralai/Devstral-Small-2505_gguf --local-dir mistralai/Devstral-Small-2505_gguf`. Then in bash terminal, run `lms import {model_name}` in the directory where you've downloaded the model checkpoint (e.g. run `lms import devstralQ4_K_M.gguf` in `mistralai/Devstral-Small-2505_gguf`)
+### 1. Install LM Studio
 
-3. Open LM Studio application, you should first switch to `power user` mode, and then open the developer tab:
+Download and install the LM Studio desktop app from [lmstudio.ai](https://lmstudio.ai/).
 
-![image](./screenshots/1_select_power_user.png)
+### 2. Download Devstral Small
 
-4. Then click `Select a model to load` on top of the application:
+1. Make sure to set the User Interface Complexity Level to "Power User", by clicking on the appropriate label at the bottom of the window.
+2. Click the "Discover" button (Magnifying Glass icon) on the left navigation bar to open the Models download page.
 
-![image](./screenshots/2_select_model.png)
+![image](./screenshots/01_lm_studio_open_model_hub.png)
 
-5. And choose the model you want to use, holding `option` on mac to enable advanced loading options:
+3. Search for the "Devstral Small 2505" model, confirm it's the official Mistral AI (mistralai) model, then proceed to download.
```

Comment:
If they are using CPU will they need to make a decision about which quantization? Or does LM Studio handle that